.github/workflows/drivers.yml

```diff
@@ -25,21 +25,10 @@ env:
   DRIVERS_BUCKET: ${{ inputs.drivers-bucket }}
 
 jobs:
-  # This sentinel job exists such that this workflow call "runs" even
-  # if subsequent driver build steps are skipped. This means that other
-  # workflows can depend on this workflow and still run even if it is skipped
-  #
-  # This only affects release branches, so is limited to those.
-  sentinel:
-    runs-on: ubuntu-latest
-    if: startsWith(github.ref_name, 'release-') || github.ref_type == 'tag'
-    steps:
-      - run: echo Drivers Build
-
   split-tasks:
     runs-on: ubuntu-latest
     if: |
-      (github.event_name == 'push' && github.ref_name == 'master') ||
+      (github.event_name == 'push' && github.ref_name == 'drivers') ||
       github.event_name == 'pull_request'
```

Comment:
We can probably get rid of this condition altogether, there's only one calling workflow and that only runs on push to drivers and PRs (I don't expect us to issue tags against this branch)